README.md

```diff
@@ -3,9 +3,8 @@
 
 Package jsonrpc2 provides a [Go](https://golang.org) implementation of [JSON-RPC 2.0](http://www.jsonrpc.org/specification).
 
-This package is **experimental** until further notice.
```

Comment:
I noticed that this line has existed since the initial commit (Oct. 2016). I assume that this was added because it predates when [Go modules became a thing in Go 1.11](https://go.dev/doc/go1.11#modules) ([Aug 2018](https://go.dev/doc/devel/release)). Now that we have Go modules that can be versioned using semantic versioning, with this library currently sitting at `v0.2.0`, I made the assumption that this line is no longer required. In other words, the fact that there is no `v1+` it's understood that this library is not considered stable.